### ISSUE TITLE
Distance Map creates gradient image if no background present

### ISSUE THREAD

**Issue by imagejan** [knime-ip/knip#465]
The *Distance Map* node, when fed an entirely white (foreground-only) image, produces a gradient image with intensities I(y) = (w+h) + y, which can lead to misleading results.

Of course, when ignoring image borders, a foreground-only image is a pretty much undefined case, but I'd expect to get a flat image rather than an image that suggests any kind of directionality.

Here's an example workflow reproducing the issue:
[Distance Map Issue.zip](https://github.com/knime-ip/knip/files/892239/Distance.Map.Issue.zip)

**Comment by gudlap** [comment]
Wouldn't adding the "Out of Bounds Strategy" (Zero-Value) to the Distance Map node actually solve this issue?

**Comment by imagejan** [comment]
Yes, allowing the user to specify the out-of-bounds strategy would definitely solve this. Also, defaulting to and `extendBorder` strategy would lead to a less surprising behavior for most use cases, I assume.
